SourceCode/SmokeTest/SmokeTestCollection.cs

```diff
@@ -74,6 +73,7 @@ public class SmokeTestCollection
 			RelativityUrl = relativityUrl;
 			InstanceSettingManager = instanceSettingManager;
 			AuditObjectManagerUiService = auditObjectManagerUiService;
+			_ipAddressForVisualStudio = ipAddressForVisualStudio; // We pass this Ip Address when running tests from Visual Studio

```

Comment:
can we mention in the comment that this ipAddress is for the instance/DevVM where the tests are running to be more specific.

Reply:
Updated with more detail